lib/wallaby/query.ex

```diff
@@ -235,13 +236,17 @@ defmodule Wallaby.Query do
     Keyword.get(conditions, :count)
   end
 
+  def getNumber(%Query{conditions: conditions}) do
+    Keyword.get(conditions, :get)
+  end
+
   def inner_text(%Query{conditions: conditions}) do
     Keyword.get(conditions, :text)
   end
 
   def result(query) do
     cond do
-      count(query) == 1 ->
+      count(query) == 1 || getNumber(query) != nil ->
```

Comment:
I know it's a small condition, but I find myself looking at this and not really understanding it until I put some solid thought into it. So, personally I'd like to extract the second condition into something like `specific_element_requested` or something? :)